doc/user-manual/getting-started/installation.rst

```diff
@@ -174,23 +174,54 @@ Nix or NixOS
 ------------
 
 Agda is part of the Nixpkgs collection that is used by
-https://nixos.org/nixos. To install Agda and agda-mode for Emacs,
-type:
+https://nixos.org/nixos. There are two ways to install Agda from nix:
 
-.. code-block:: bash
+* The new way: If you are tracking ``nixos-unstable`` or
+  ``nixpkgs-unstable`` (the default on MacOS) or you are using NixOS
+  version 20.09 or above then you should be able to install Agda (and
+  the standard library) via:
+
+    .. code-block:: bash
+
+    nix-env -f "<nixpkgs>" -iA agda
+    agda-mode setup
+    echo "standard-library" > ~/.agda/defaults
```

Comment:
This won't install the standard library, you would need `agda.withPackages (p: [ p.standard-library ])`

Reply:
cut and paste fail. fixed.

Reply:
> cut and paste fail. fixed.

differently broken :(, I'll try to fix it.